src/message/messageActionSheet.js

```diff
@@ -165,6 +165,10 @@ const shareMessage = ({ message }) => {
   });
 };
 
+const openUrl = ({ message }) => {
+  console.log(message.content);
```

Comment:
Hi @pulkonet, where is this `openUrl()`method will be used?

Also, you may work with `console.log()`s in your dev environment, but in PR commits try not to add them. Instead, you may type `git stash` to add the methods for debugging, to the stash. After sending the commit, you can bring back the contents of last saved stash with `git stash pop`.

Reply:
Oh. You're right. I added this for testing something and forgot to remove this. Thanks.